inlong-sdk/dataproxy-sdk-twins/dataproxy-sdk-python/README.md

```diff
@@ -39,3 +39,57 @@ chmod +x ./build.sh
+| retry_num                | 3                                                                  | maximum resend times                                                                                       |
+| max_active_proxy         | 3                                                                  | maximum number of established connections with dataproxy                                                   |
+| max_buf_pool             | 50 `*`1024`*` 1024                                                 | byte, the size of buffer pool                                                                              |
+| log_num                  | 10                                                                 | maximum number of log files                                                                                |
+| log_size                 | 10                                                                 | MB, maximum size of one log file                                                                           |
+| log_level                | 2                                                                  | log level: trace(4)>debug(3)>info(2)>warn(1)>error(0)                                                      |
+| log_file_type            | 2                                                                  | type of log output: 2->file, 1->console                                                                    |
+| log_path                 | ./logs/                                                            | log path                                                                                                   |
+| proxy_update_interval    | 10                                                                 | interval of requesting and updating dataproxy lists from manager                                           |
+| manager_url              | "http://127.0.0.1:8083/inlong/manager/openapi/dataproxy/getIpList" | the url of manager openapi                                                                                 |
+| need_auth                | false                                                              | whether need authentication while interacting with manager                                                 |
+| auth_id                  | ""                                                                 | authenticate id if need authentication                                                                     |
+| auth_key                 | ""                                                                 | authenticate key if need authentication                                                                    |
+
+## Usage
+
+1. First, init dataproxy-sdk:`init_api(config_file)`. Here, `config_file` is the path of your config file, and absolute
```

Comment:
Please describe the steps of use in complete English sentences.

Reply:
The description has been optimized.